src/main/core-api/java/com/mysql/cj/conf/PropertyDefinitions.java

```diff
@@ -665,6 +665,9 @@
                 new BooleanPropertyDefinition(PropertyKey.allowXmlUnsafeExternalEntity, DEFAULT_VALUE_FALSE, RUNTIME_NOT_MODIFIABLE,
                         Messages.getString("ConnectionProperties.allowXmlUnsafeExternalEntity"), "0.2.0", CATEGORY_SECURITY, Integer.MAX_VALUE),
 
+                new BooleanPropertyDefinition(PropertyKey.useAwsIam, DEFAULT_VALUE_FALSE, RUNTIME_NOT_MODIFIABLE,
+                        Messages.getString("ConnectionProperties.useAwsIam"), "0.2.0", CATEGORY_SECURITY, Integer.MAX_VALUE),
```

Comment:
I think the version here should be `0.3.0`. @sergiyvamz what do you think?

Reply:
I believe the latest is 0.2.1. We can bump up version to "0.2.2" here for parameters and for driver itself.